content/en/docs/collector/management.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Management
+weight: 23
+---
+
+Please be sure to review the following documentation as prerequisite:
```

Comment:
```suggestion
This document describes how you can manage your OpenTelemetry collector
deployment at scale.

To get the most out of this page you should know how to install and configure
the collector. These topics are covered elsewhere:
```

Reply:
Hey @mhausenblas. This had been marked as resolved, but the changes were not applied AFAICT.

Reply:
@mhausenblas - btw, in case you didn't know: you can use the "commit" buttons that are a part of the suggestions to accept changes. It makes is easy, and less error prone, to apply changes.